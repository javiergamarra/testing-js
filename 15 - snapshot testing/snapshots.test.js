describe("snapshots capturan el valor inicial", () => {
  const something = { key: "valu" };

  it("guardamos el valor", () => {
    expect(something).toMatchSnapshot();
    something.nuevaKey = "nuevoVal2or";
    expect(something).toMatchSnapshot();
  });

  // it("mutamos el valor", () => {
  //   expect(something).toMatchSnapshot();
  // });
});
