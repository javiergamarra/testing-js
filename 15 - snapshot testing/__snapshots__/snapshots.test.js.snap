// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots capturan el valor inicial guardamos el valor 1`] = `
Object {
  "key": "valu",
}
`;

exports[`snapshots capturan el valor inicial guardamos el valor 2`] = `
Object {
  "key": "valu",
  "nuevaKey": "nuevoVal2or",
}
`;
